static/js/publisher/metrics/metrics.js

```diff
@@ -12,6 +12,10 @@ function renderMetrics(metrics) {
     return false;
   }
 
+  if (d4) {
```

Comment:
Awesome code

Reply:
just getting ready for when d4 is released and replaces d3

Reply:
? I'm so confused. Surely we're not merging that?

Reply:
Of course not... But we need to create an error to ensure our errors are still coming through.... Once QA is done I'll remove this code

Reply:
More details on ^ and because we white list hosts, I've white listed the demo url for QAing